src/redux/books/book.js

```diff
@@ -0,0 +1,24 @@
+const BOOK_REMOVED = 'BOOK_REMOVED';
+const BOOK_ADDED = 'BOOK_ADDED';
```

Comment:
- [ ] Kindly the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/210825276-9a217a98-23e4-4975-82f5-2d8295c0a14d.png)